sys/auto_init/netif/auto_init_encx24j600.c

```diff
@@ -44,7 +44,11 @@ void auto_init_encx24j600(void)
 {
     DEBUG("auto_init_encx24j600(): initializing device...\n");
     /* setup netdev2 device */
-    encx24j600_setup(&encx24j600, ENCX24J600_SPI, ENCX24J600_CS, ENCX24J600_INT);
+    encx24j600_params_t p;
+    p.spi       = ENCX24J600_SPI;
+    p.cs_pin    = ENCX24J600_CS;
+    p.int_pin   = ENCX24J600_INT;
+    encx24j600_setup(&encx24j600, &p);
```

Comment:
Unrelated, but the param struct should be board-defined here too, IMHO (unless I'm overlooking something).